.github/workflows/auditbeat-macos-unit-tests.yml

```diff
@@ -0,0 +1,55 @@
+name: auditbeat-macos-unit-tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+    paths:
+      - auditbeat/
+      # OSS
+      - go.mod
+      - pytest.ini
+      - dev-tools/
+      - libbeat/**
+      - testing/**
```

Comment:
same, I think this is not needed for the unit-tests, as far as I see at https://github.com/elastic/beats/tree/main/testing/environments